### ISSUE TITLE
[v21] Really difficult speedrunner seed?

### ISSUE THREAD

**Issue by Dessyreqt** [Dessyreqt/smrandomizer#20]
Submitter: BluePlasmaSpark
[v21] S3536114. I am either seriously overlooking something or the seed is very difficult or near impossible. It is on speedrunner difficulty, not masochist. The seed forces the player to find items in locations that are difficult to obtain they are

Gravity Suit, Space Jump, Hi-Jump Boots, Speed Booster(?)

Here is a report for why each item does not work, and why it is too difficult to complete the seed:
A) Gravity suit- It forces the player through Sponge Bath without Gravity, Space Jump, Hi-Jump, or Speed Booster
B) Space Jump- Forces the player to do a partially suitless Maridia
C) Hi-Jump and Speed Booster- They are both in Lower Norfair, making the player enter LN without Gravity, Speedbooster, or Hi-Jump. 
If any of the following items are obtainable in any other way, the seed would be fine
         
These tricks make the seed very difficult for the difficulty setting. I am not sure if this qualifies as a bug, but thank you for taking the time to look through this.

**Comment by moriwizz** [comment]
Yeah, this seed is practically impossible. It seems like the only possibility besides X-Ray climbing is to do some near impossible or actually impossible crab freezing to make it up the first Maridia room to get Space Jump and from there go to the Wrecked Ship to get Gravity Suit.
